pkg/controller/import-controller.go

```diff
@@ -316,7 +349,7 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 
 	requiresScratch := r.requiresScratchSpace(pvc)
 	if requiresScratch {
-		name := scratchNameFromPvc(pvc)
+		name := createScratchNameFromPvc(pvc)
```

Comment:
scratch pvc name not stored in annotation?

Reply:
Scratch PVC name is created once, and it is here at this line. Then this name is available as a ClaimName on ScratchVolName = "cdi-scratch-vol" on pod, so we can always get the correct name. And we do not need to get scratch name in other places, so I wonder if we need to store it anywhere.